pep-0684.rst

```diff
@@ -0,0 +1,640 @@
+PEP: 684
+Title: A Per-Interpreter GIL
+Author: Eric Snow <ericsnowcurrently@gmail.com>
+Discussions-To: python-dev@python.org
```

Comment:
If you've already posted a thread on this, as your `Post-History` would imply, please link the thread directly here instead. Otherwise, make sure to remember to come back and update this as soon as you do. Suggestion as an example:

```suggestion
Discussions-To: https://example.com/thread_1
```